packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md

```diff
@@ -0,0 +1,180 @@
+
+6. Leave the **Authentication field** and **Authentication secret** values blank in this example. However, to add Basic Authentication, add the application code required at [Add Basic Authorization and Body Parsing](/docs/guides/common-hook-set-up-steps/nodejs/main/#add-basic-authorization-and-body-parsing) and then enter the following values for those fields:
+
+    * **Authentication field** = `authorization`
+
+    * **Authentication secret** = `Basic YWRtaW46c3VwZXJzZWNyZXQ=`
+
+7. In the **REQUESTS** section of the dialog box, subscribe to the Event Type you want to monitor. In this example, a user created in the Okta org: `User created`.
+
+8. Click **Save & Continue**.
+
+9. With your Hookdeck session and local application running, complete the one-time verification Okta call at this time or verify the Event Hook later.
+
+### Verify the Event Hook
+
+You must verify the Event Hook to prove that your external service controls the endpoint. See [One-Time Verification Request](/docs/concepts/event-hooks/#one-time-verification-request).
+
+To complete the one-time verification of the Event Hook:
+
+* After creating the Event Hook, and if your Hookdeck session and local application are ready to handle the request, click **Verify** to complete the one-time verification step.
+
+or
+
+* After making sure that your Hookdeck session and local application are ready for the external verification call, go to the Event Hooks table, click the **Actions** drop-down menu of your **UNVERIFIED** Event Hook, and select **Verify**.
+
+The Event Hook is now set up with a status of **VERIFIED** and is ready to send Event Hook calls to your external service.
+
+## Preview, test, and review the Event Hook
+
+With your local application now exposed externally through a Hookdeck session, you can preview and test Okta Event Hook calls, as well as review details of the calls using the Hookdeck dashboard. The Okta org is also set up to call your local application when an event is triggered. In this example, the event triggers when a user is added to your Okta org.
+
+### Preview
+
+To run a preview call of your Event Hook:
+
+1. In the Admin Console, go to **Workflow** > **Event Hooks**.
+1. Locate the Event Hook you created during the set-up step. In this example, select `New User Event Hook` or the name you gave the Event Hook.
+1. Click the **Actions** menu for this hook, and select **Preview**.
+1. In the **Configure Event Hook request** section, select an event from the **Event Type** drop-down menu. In this example, there is only one: `User Created (user.lifecycle.create)`.
+1. The most recent event populates the **Preview & Deliver Event Hook** section with the JSON body of the Event Hook if there is one. You can also select an older event from the **System Log Event** drop-down menu. If no event is available, the JSON body populates with sample data.
```

Comment:
The instructions to select an older even from the **System Log Event** is not clear - maybe you can show me how this part works?

Reply:
Maybe over-explanation again. If, for example, you've added 4 or 5 people, those 4 or 5 events appear in the drop-down menu, and you can choose among them for the preview JSON. I'm going to remove that content.